kivymd/uix/tab.py

```diff
@@ -481,7 +520,7 @@ def switch_tab(self):
 from kivy.uix.widget import Widget
 from kivy.utils import boundary
 
-from kivymd.font_definitions import fonts
+from kivymd.font_definitions import fonts, theme_font_styles
```

Comment:
Well, as far as i know, when a dic is imported. The `__import__` method only creates a link to the pointer, and add the value to the global environment rather to contain it in the file/lib. Because of this, you could alter the value in any file and every change should be shared. Yet, I'm not certain if the event dispatcher would call any event to it.

Also, as far as I know, comparing a value to a list were always faster than to a dict.

![Captura de pantalla de 2021-02-23 17-38-17](https://user-images.githubusercontent.com/5509325/108923182-fb67ed80-75fd-11eb-82a6-e337e8514d38.png)
![Captura de pantalla de 2021-02-23 17-37-22](https://user-images.githubusercontent.com/5509325/108923185-fc991a80-75fd-11eb-80dc-600bd63df2c4.png)
![Captura de pantalla de 2021-02-23 17-37-33](https://user-images.githubusercontent.com/5509325/108923190-fe62de00-75fd-11eb-9438-ccf5b8bdf083.png)
![Captura de pantalla de 2021-02-23 17-39-16](https://user-images.githubusercontent.com/5509325/108923249-163a6200-75fe-11eb-8725-e8788851924f.png)

Reply:
The last commit ensures parity between the pointer and the theme_cls object.

Reply:
To me, you are adding complexity on top of complexity and adding a lot of new code and possible error modes for the relatively small performance gain for getting the list of values from a list rather than dict items. 

The class `ThemeManager` HAS an existing attribute that holds the current font styles. Why not just always drive from the current value of that attribute rather than require another value to be kept up to date?

If I was technical lead on this project I would say this solution adds to the error vectors of the project and makes the project less maintainable - and indeed would probably not give this change "readability' approval. But I think this will work.